test/internal/sync/sync.go

```diff
@@ -0,0 +1,29 @@
+// This package holds common synchronization primitives used in test writing
+package sync
```

Comment:
why do we need this whole new `NewLazyString` type here? Why can't we just have an init function or a variable set to the result of a do once?

Reply:
if we did a one off, we still would have a bare `once` lying around, which feels prone to misuse, so the `LazyString` keeps everything contained.
The init would always be run, regardless of if LCOW uVM are ever used, so that felt unnecessary.
Though I do agree that this is likely overkill, especially if its not reused elsewhere  ...

Reply:
I removed it, since I couldnt think of anywhere else that would benefit from it, and switched to an init